### DISCUSSION TITLE
Why does the online editor rename content folders?

### DISCUSSION THREAD

**Discussion by BsNoSi** [datenstrom/community#425] [Ask a question] [answered]
I want to pull up #112 but giving  a more precise description of the things that happen in V0.8.4 (and before).

See following folder structure

    1-home
    2-wiki
       page.md
    3-impress

Opening page.md with the online editor and changing 

    title: This is the page

to

    title: A new Title
    published: 2019-05-06

will  lead to this folder structure:

    1-home
    3-impress
    2019-05-06-a-new-title
       page.md

Without additional `published` or `modified` it leads to

    1-home
    2-a-new-title
       page.md
    3-impress

Both lead to "file not found" for links into this folder.

You can simply prevent this by editing the header area of "page.md" files not online.

 I think it is not worth the time to fix this but I think it would be good to know it. Maybe it should be added as a hint in the online manuals.

**Comment by markseuffert** [comment, marked as answer]
Here are some tips for the [online editor](https://github.com/annaesvensson/yellow-edit):

1. Do not change the `Title`, if you don't want to rename a page. 
2. Do use `TitleSlug`, if you want a fixed filename/URL regardless of title.
3. You can edit the content of a page, it will not change filename/URL.

Sometimes it's hard to balance the features of a product, you don't want too less or too many. I suspect that the problem with the current editor comes from being used to an old version of the editor, which did not rename folders. The current editor makes no difference between files and folders any more. 

If more people want to change the current behaviour/documentation, please let us know.
